pkg/scheduler/api/job_info.go

```diff
@@ -371,6 +402,22 @@ func (ji *JobInfo) FitError() string {
 	return reasonMsg
 }
 
+// GetBackillReadiness checks whether the job is ready or ready
+// using backill job's resources ConditionallyReady means the
+// job is ready but some resource is used by backfilled job
+func (ji *JobInfo) BackfillReady() bool {
+	allocatedTasks := ji.GetTasks(AllocatedStatuses()...)
+	allocatedTasksCnt := int32(len(allocatedTasks))
+
+	allocatedOverBackfillTasks := ji.GetTasks(Borrowing)
+	allocatedOverBackfillTasksCnt := int32(len(allocatedOverBackfillTasks))
+	if allocatedTasksCnt+allocatedOverBackfillTasksCnt >= ji.MinAvailable {
```

Comment:
we should not check `minAvailable` in Job, `minAvailable` is used by gang plugin. I need delegate to gang plugin to do that. So when we disable gang plugin, minAvailable will not work, refer to `JobReady`.

Reply:
WIP